packages/rocketchat-message-snippet/server/startup/settings.js

```diff
@@ -0,0 +1,13 @@
+Meteor.startup(function() {
+	RocketChat.settings.add('Message_AllowSnippeting', true, {
```

Comment:
We usually start new features as disabled by default then users can enable and test, can you change to `false`?